bindings/python/CHANGELOG.md

```diff
@@ -6,10 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.11.0]
 
+### Fixed
+
+- [#585] Conda version should now work on old CentOS
```

Comment:
You also need to add links at the end of this file

Reply:
Spoiled by GH usual autolinks :)